host/sgx/create.c

```diff
@@ -216,6 +213,15 @@ static oe_result_t _add_control_pages(
          */
         tcs->fsbase = *vaddr + (5 * OE_PAGE_SIZE);
```

Comment:
The 4th page is for thread local storage(previous version of td is patched to one page by putting thread local storage to the end), and it can only be accessed in the format of fs:-offset, thus fs must point to *vaddr + (5 * OE_PAGE_SIZE).